style.css

```diff
@@ -898,3 +899,27 @@ h3 {
   width: 150%;
   height: 150%;
 }
+
+/* Mobile-Menu */
+
+.mobileMenu {
+  color: #3a4a42;
+  flex-direction: column;
+  display: none;
+}
+
+.mobileMenu a {
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #3a4a42;
+  margin: 3%;
+}
+
+.mobileMenu a:link {
+  text-decoration: none;
+}
+
+.btMob {
+  all: unset;
+}
```

Comment:
As requested on the project requirements: 
![image](https://user-images.githubusercontent.com/42799579/208773295-a341e8d7-59d2-4e3f-93d7-edce7adcb9b9.png)

Please make sure to follow the origin design as much as possible 😉. To achieve this, kindly adjust your close (X) button to the right of the mobile menu.
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/208773454-591912f4-20e3-4172-965d-37c32941a091.png)  | ![image](https://user-images.githubusercontent.com/42799579/208773422-16ab7d8c-1786-451d-abc6-e1064dcecf7f.png) |

Great job with the menu functionality! 😄